include/util/rectangle.hpp

```diff
@@ -168,6 +167,18 @@ struct RectangleInt2D
                min_lat != FixedLatitude{std::numeric_limits<std::int32_t>::max()} &&
                max_lat != FixedLatitude{std::numeric_limits<std::int32_t>::min()};
     }
+
+    static RectangleInt2D ExpandMeters(const Coordinate &coordinate, const double meters)
```

Comment:
It is basically re-implemented function from Valhalla: https://github.com/valhalla/valhalla/blob/c14d3ac8e4745a17e5e99a50dac32c838fc3475b/valhalla/midgard/util.h#L107